gradle/publishing.gradle.kts

```diff
@@ -0,0 +1,50 @@
+apply(plugin = "maven-publish")
+apply(plugin = "signing")
+
+afterEvaluate {
+    configure<PublishingExtension> {
+        publications {
+            create<MavenPublication>("maven") {
+                from(components[componentName()])
+                pom {
+                    name.set("${project.group}:${project.name}")
+                    description.set(Config.Sentry.description)
+                    url.set(Config.Sentry.repository)
+                    packaging = packaging()
+                    licenses {
+                        license {
+                            name.set(Config.Sentry.licence)
+                            url.set(Config.Sentry.licenceUrl)
+                        }
+                    }
+                    developers {
+                        developer {
+                            id.set(Config.Sentry.userOrg)
+                            email.set(Config.Sentry.devEmail)
+                            name.set(Config.Sentry.devName)
+                        }
+                    }
+                    scm {
+                        connection.set(Config.Sentry.scmConnection)
+                        developerConnection.set(Config.Sentry.scmDevConnection)
+                        url.set(Config.Sentry.scmUrl)
+                    }
+                    issueManagement {
+                        url.set(Config.Sentry.issueTracker)
+                    }
+                }
+                groupId = project.group.toString()
+                artifactId = project.name
+            }
+        }
+
+//        configure<SigningExtension> {
+//            sign(publications["maven"])
+//        }
```

Comment:
can we check if it's a CI build and ignore it though? we can do this thru GH system envs iirc.

Reply:
https://docs.github.com/en/actions/reference/environment-variables#default-environment-variables
`CI` probably

Reply:
I added a property "signingEnabled". I think it makes sense to have it decoupled from CI as I also would not like to sign artifacts when i just build them locally.